app/views/miq_policy/_event_details.html.haml

```diff
@@ -10,7 +10,7 @@
             %label.control-label.col-md-2= _("Description")
             .col-md-10
               %p.form-control-static
-                %img{:src => image_path("100/event-#{h(@event.name)}.png")}
+                %i{:class => @event.decorate.fonticon}
```

Comment:
@cben Thanks for noticing, removed in https://github.com/ManageIQ/manageiq-ui-classic/pull/194